src/extensions/default/JavaScriptQuickEdit/unittest-files/jquery-ui/themes/base/jquery.ui.spinner.css

```diff
@@ -10,7 +10,7 @@
 .ui-spinner { position:relative; display: inline-block; overflow: hidden; padding: 0; vertical-align: middle; }
 .ui-spinner-input { border: none; background: none; padding: 0; margin: .2em 0; vertical-align: middle; margin-left: .4em; margin-right: 22px; }
 .ui-spinner-button { width: 16px; height: 50%; font-size: .5em; padding: 0; margin: 0; z-index: 100; text-align: center; position: absolute; cursor: default; display: block; overflow: hidden; right: 0; }
-.ui-spinner a.ui-spinner-button { border-top: none; border-bottom: none; border-right: none; } /* more specificity required here to overide default borders */
+.ui-spinner a.ui-spinner-button { border-top: none; border-bottom: none; border-right: none; } /* more specificity required here to override default borders */
```

Comment:
jQueryUI is a third party addon so the changes should be reported in https://github.com/jquery/jquery-ui. Can you remove this change

Reply:
@petetnt 
I posted a new PR as you suggested. I would appreciate it if you confirm.
Thank you.

Reply:
@petetnt
I fixed the same parts the same way. However, I saw the phrase "All checks have failed" in # 13853.
Can you see what the problem is with this?

Reply:
Hi @JungGaBin,

you'll need to sign our Contributors License Agreement (CLA) http://dev.brackets.io/brackets-contributor-license-agreement.html

Reply:
@petetnt 
I signed it!